src/prefect/cli/shell.py

```diff
@@ -129,7 +132,7 @@ async def watch(
     ),
     flow_run_name: str = typer.Option(None, help="Name of the flow run."),
     flow_name: str = typer.Option("Shell Command", help="Name of the flow."),
-    stream_output: bool = typer.Option(True, help="Stream the output of the command."),
+    stream_stdout: bool = typer.Option(True, help="Stream the output of the command."),
```

Comment:
Generally if we know we're going to make breaking changes like this (and may in the future), I think we should have or should put an experimental decorator on this CLI. 

and / or

deprecate these fields instead of replace them.

Reply:
I opted to change it instead of deprecate as this is the first release the CLI would be available in.

Reply:
oh awesome, thought it was already out in the world. cool